client/python/cryoet_data_portal/src/cryoet_data_portal/templates/Model.jinja2

```diff
@@ -43,14 +43,14 @@ class {{ model.name }}(Model):
     find.__func__.__doc__ = Model.find.__func__.__doc__ + find.__func__.__doc__
 
     @classmethod
-    def get_by_id(cls, **kwargs):
+    def get_by_id(cls, *args, **kwargs):
```

Comment:
Not a blocker, but is there any way we can retain the `client, id` signature here?

It's a bit concerning to me that in the process of improving the web docs we're making IDE-based code inspection worse.

Reply:
Fair enough - I've updated this PR to make the method signatures more explicit. Mind taking another look?

Reply:
Yep, this works for me in PyCharm.